docs/rst/commands/odgi_flip.rst

```diff
@@ -0,0 +1,65 @@
+.. _odgi flip:
+
+#########
+odgi flip
+#########
+
+Flip (reverse complement) paths to match the graph.
+
+SYNOPSIS
+========
+
+**odgi flip** [**-i, --input**\ =\ *FILE*] [*OPTION*]…
+
+DESCRIPTION
+===========
+
+The odgi flip command converts BED records against graph paths into new paths labeled by the BED record name.
```

Comment:
This is about BED records, but I thought the tool just flips all paths? I am confused here.